### DISCUSSION TITLE
can i configure keycloak authentication with refine

### DISCUSSION THREAD

**Discussion by subhashnew** [refinedev/refine#1574] [Q&A] [answered]
How can i use the keycloak authentication for refine-react application.Can someone give a brief explain.

**Comment by omeraplak** [comment, marked as answer]
Hey @subhashnew ,
I have not worked with Keycloak before, but when I looked at the examples, I realized that it is possible.

Have you looked at this example?
https://github.com/react-keycloak/react-keycloak-examples/tree/master/examples/react-router

If you start integrating and get stuck we are always here to help! 
BTW, We are waiting for you on our Discord server! -> https://discord.gg/UuU3XCc3J5

**Reply by subhashnew** [reply]
Yes i went through that example i also thought it should be possible.Thank you very much.I will be ask if there is a update.
